src/modules/tests/test_data/test-data.reducer.ts

```diff
@@ -119,6 +120,22 @@ export function testDataReducer(
           physical: !state.ETA.physical,
         },
       };
+    case testDataActions.TOGGLE_CONTROL_ECO:
```

Comment:
You don't seem to toggle if eco has been completed or not - is this needed?

Completed can be enabled on it's own or when one of the faults are triggered

Reply:
Good spot, forgot about that

Reply:
Will have to mark the ECO fault 'completed' field in the test report page